wishlist.css

```diff
@@ -0,0 +1,82 @@
+body{background-color: white;
+    color : black;
+    font-family: "comic sans ms";
+    
+}
+.container{
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    align-items: center;
```

Comment:
You can extract this in a utility class so that it can be used anywhere else as well